### PR TITLE
Bump debian/changelog for 10.2 series

### PR DESCRIPTION
**Pull request by ottok**

### CODE REVIEW

**Comment by svoj** [comment]
@ottok, what are we supposed to do with this PR?

**Comment by ottok** [comment]
Merge it?

File: debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+mariadb-10.1 (10.2.0) unstabkke; urgency=low
```
Comment (jocel1) [inline debian/changelog]: should be mariadb-10.2 and "unstable" here ? :)

**Comment by svoj** [comment]
Don't you have write access to MariaDB repository? Normally I'd assign Debian contributions to you. Do you want somebody to review this patch?

**Comment by ottok** [comment]
I do have direct access but I prefer a work-flow where somebody reviews (at least a bit) before the commit goes to main branch due to QA reasons.

@jocel1 Thanks for spotting the typo. I now force-pushed to this branch a fixed commit.

**Comment by svoj** [comment]
Merged, thanks.

**Comment by ottok** [comment]
@svoj You only pulled this locally? Good, because I noticed another typo and force pushed now commit 5da4436c3161595d544a0c04bdb52edab4e13c3b Please force pull it if you only have it on a local branch.

Previously the 10.2 master branch didn't even build, but since last weeks fixes it now does and I can make test builds and actually notice my own typos.

**Comment by svoj** [comment]
@ottok it's in main branch: https://github.com/MariaDB/server/commit/b7eaba474dc6356d39c42ea5af73476a76294fe9

I fixed that typo.

**Pull request closed without merging**